sdks/python/apache_beam/io/textio_test.py

```diff
@@ -53,6 +53,16 @@
 from apache_beam.transforms.core import Create
 
 
+def write_data(*args, **kwargs):
```

Comment:
It is hard to reason about this helper in the context of the test scenarios. Let's copy the existing implementation of write_data into this test suite, modify it to return strings as second tuple argument, and add a docstring explaining the arguments and the type of result.

This would make it easier for these helpers evolve separately. For example, for the purposes of textio_test, I think it makes sense to allow customizing payload instead of default 'line', and adding more interesting testcases of unicode strings in the future.

Reply:
Good point. I copied the implementation and added documentation in both files.